index.html

```diff
@@ -224,6 +224,36 @@ <h3 class="connect-text">LET'S CONNECT</h3>
       </div>
     </div>
   </section>
+
+  <section class="contact-form">
+    <div class="contact-form-holder">
+      <h2 class="contact-title">Contact me</h2>
+
+      <p class="p-pargraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form action="https://formspree.io/f/xqkwonvv" class="form-contact" method="post">
+        <ul class="ul-form">
+          <li class="li-form">
+            <input type="text" placeholder="Emmanue Hagan" name="name" id="name_form" maxlength="30" required />
+          </li>
+          <li class="li-form">
+            <input type="email" placeholder="haganemmanuel@gmail.com" name="email" id="email_form" required />
+          </li>
+          <li class="li-form">
+            <textarea name="message" placeholder="Write your message here" id="message" cols="30" rows="10"
+              maxlength="500" required></textarea>
+          </li>
+          <li class="li-form">
+            <input id="get-submit" type="submit" value="Get It Touch" class="btn-in" />
```

Comment:
- [ ] I suggest sticking to the Figma design so it would be better if you change the button like the one in the Figma design 
- [ ] Figma design
![image](https://user-images.githubusercontent.com/61790587/166972481-aeb2c088-8ac2-4f83-8f13-fec542b9be42.png)

- [ ] Yours
![image](https://user-images.githubusercontent.com/61790587/166972532-5bff9574-27ab-415f-85ee-02bc798dfad8.png)

- [ ] And also don't forget to add style for hover and active 
![image](https://user-images.githubusercontent.com/61790587/166972593-ece7e52e-b092-4a39-82be-4bf8b67843af.png)

Reply:
- [ ] @hagan199  Great work implementing the changes however,  your hover style color is different from Figma design, kindly change it as requested by the previous reviewer.